include/Surelog/API/SLAPI.h

```diff
@@ -102,47 +102,47 @@ std::vector<antlr4::ParserRuleContext*> SLgetChildrenContext(
     SV3_1aPythonListener* prog, antlr4::ParserRuleContext* context);
 
 /* Parser API */
-NodeId SLgetRootNode(FileContent* fC);
+RawNodeId SLgetRootNode(FileContent* fC);
```

Comment:
(in a future change, these could probably all take a `const FileContent*` parameter. But not current scope)